lib/panel/index.js

```diff
@@ -6,7 +6,7 @@ import PanelDock from './dock'
 import type { LinterMessage } from '../types'
 
 class Panel {
-  panel: ?PanelDock;
+  panel: PanelDock | null;
```

Comment:
@steelbrain I suspect this should be `|| null` instead of `| null`.

Reply:
It's flowtype and not JS so this is actually correct :) see https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBjOA7AzgFzFzgFsBTfACwEtsBzALjGwFcYYwAfZlkgIzIAnMAF4wAJlRA